.github/workflows/tutorials.yml

```diff
@@ -0,0 +1,79 @@
+name: Jupyter Notebooks
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tutorials:
+    name: Test Jupyter notebooks on ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+
+    env:
+      DEVITO_ARCH: "${{ matrix.compiler }}"
+      DEVITO_OPENMP: 1
+      DEVITO_BACKEND: "core"
+      PYTHON_VERSION: "3.7"
+
+    strategy:
+      # Prevent all build to stop if a single one fails
+      fail-fast: false
+      matrix:
+        name: [
+          ubuntu-16.04-gcc-py37,
+          macOS-10.14-gcc-py37
+          ]
+
+        include:
+          - name: ubuntu-16.04-gcc-py37
+            os: ubuntu-16.04
+            compiler: gcc-7
+
+          - name: macOS-10.14-gcc-py37
```

Comment:
Just to note, it was discussed that many users develop on macOS and often use `clang` as their compiler. Perhaps it would be desirable to test `gcc` on `ubuntu` and test `clang` on `macOS`

At the moment only `gcc-7` and `gcc-9` are tested.

Reply:
Will add a clang build

Reply:
Addec clang